layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -11,14 +11,10 @@ SRC_URI:append = " \
     file://uEnv.txt_internal \
 "
 
-SSTATE_ALLOW_OVERLAP_FILES += "${DEPLOYDIR}/extra_uEnv.txt"
-SSTATE_ALLOW_OVERLAP_FILES += "${DEPLOYDIR}/uEnv.txt_internal"
+SSTATE_ALLOW_OVERLAP_FILES += "${DEPLOY_DIR_IMAGE}/extra_uEnv.txt"
```

Comment:
This is interesting, I ended up removing one of them... the question is: how do you know if it is OK to have them overlapped? What if one of them has different content? (on the bbai64 case the files where empty on both cases).

Reply:
I needed to add this because my build kept failing. The beagleplay has a 'BBMULTICONFIG +=' that makes behave a bit strangely. I am sure this can be fixed in a nicer way.

Reply:
It probably features two architectures like the AI64